src/lib/gtagProvider.tsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-console */
+import { GoogleTagManager } from '@next/third-parties/google';
+
+const GtagProvider: React.FC = ({}) => {
+  const isProd = process.env.NODE_ENV === 'production';
+
+  return (
+    <GoogleTagManager
+      gtmId={'GTM-NNCH6SSL'}
```

Comment:
I know they are public but i kinda agree with the rabbit here and it'd be nice to put them in an env file, even if it was just the public .env